src/NITTA/Model/Networks/Bus.hs

```diff
@@ -73,8 +85,18 @@ data BusNetwork tag v x t = BusNetwork
       bnSignalBusWidth :: Int
     , ioSync :: IOSynchronization
     , bnEnv :: UnitEnv (BusNetwork tag v x t)
+    , -- |Set of the PUs that could be added to the network during synthesis process
+      bnPUPrototypes :: M.Map tag (PUPrototype v x t)
     }
 
+data PUPrototype v x t where
```

Comment:
Add documentation on tag formation and interpretation.

Reply:
done